src/load_map.cpp

```diff
@@ -1180,6 +1180,8 @@ void map_parser::parse_shield_symbolizer(rule & rule, xml_node const& node)
         put(sym, keys::file , parse_path(file));
         optional<halo_rasterizer_e> halo_rasterizer_ = node.get_opt_attr<halo_rasterizer_e>("halo-rasterizer");
         if (halo_rasterizer_) put(sym, keys::halo_rasterizer, halo_rasterizer_enum(*halo_rasterizer_));
+        set_symbolizer_property<symbolizer_base,composite_mode_e>(sym, keys::halo_comp_op, node);
+        set_symbolizer_property<symbolizer_base,transform_type>(sym, keys::halo_transform, node);
```

Comment:
Yes, it's not directly related to my request. And I did not really add any new functionality. The ```halo_comp_p``` and ```halo_transform``` properties were already implemented and wired to the TextSymbolizer, all what I did was to enable them for ShieldSymbolizer as well.
However, I reverted the change and will create a separate pull request.